manifest-build-tools/application/update_manifest.py

```diff
@@ -0,0 +1,442 @@
+#!/usr/bin/env python
+
+# Copyright 2015, EMC, Inc.
+
+"""
+Usage:
+./on-tools/manifest-build-tools/HWIMO-BUILD on-tools/manifest-build-tools/application/update_manifest.py \
+--repo "$url" \
+--branch "$branch" \
+--commit "$id" \
+--manifest_download_url https://dl.bintray.com/rackhd-mirror/binary/manifest/ \
+--git-credential https://github.com,GITHUB_CREDS \
```

Comment:
if the existing functions ```repo_operator.setup_gitbit```  and ```Manifest(path_name, self.__git_credentials)``` needs git-credential during initialization ,  then :+1: now.
but I still prefer it to be optional , + @PengTian0   to be aware